### PR TITLE
post assigning updated & README

### PR DESCRIPTION
**Pull request by pkgodara**

### CODE REVIEW

File: includes/AssignPost.php [hunk 1]
```diff
@@ -98,7 +98,7 @@
 echo "<table>";
 while($row = mysqli_fetch_row($res))
 {
-	echo "<tr><td><h2>$row[0] :</h2></td><td> <input type='text' name=$row[0] autofocus ></td></tr> <br>";
+	echo "<tr><td><h2>$row[0] :</h2></td><td> <input type='text' name=$row[0] value=$row[1] autofocus ></td></tr> <br>";
```
Comment (pkgodara) [inline includes/AssignPost.php:101]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 2]
```diff
@@ -159,7 +159,26 @@
 		echo "<br><br><h3 style = 'color:white ;'> $post Post  already exists .</h3>";
 		die ();
 	}
-
+
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:162]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 3]
```diff
@@ -159,7 +159,26 @@
 		echo "<br><br><h3 style = 'color:white ;'> $post Post  already exists .</h3>";
 		die ();
 	}
-
+	
+	// create table for asigning posts
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:163]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 4]
```diff
@@ -159,7 +159,26 @@
...
+	$stmt = $sqlConn->prepare("CREATE TABLE IF NOT EXISTS $assignPostTable ($postTitle VARCHAR(100) NOT NULL, $assignedUser VARCHAR(100), INDEX idx USING BTREE ($postTitle))");
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:164]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line exceeds 120 characters; contains 176 characters

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 5]
```diff
@@ -159,7 +159,26 @@
...
+	if ( ! $stmt->execute() )
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:165]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected 0 spaces, found 4
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 5

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 6]
```diff
@@ -159,7 +159,26 @@
...
+	{
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:166]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 7]
```diff
@@ -159,7 +159,26 @@
...
+		echo"there is a problem with database<br>";
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:167]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 8]
```diff
@@ -159,7 +159,26 @@
...
+		die ();
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:168]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 9]
```diff
@@ -159,7 +159,26 @@
...
+	}
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:169]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected 0 spaces, found 4

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 10]
```diff
@@ -159,7 +159,26 @@
...
+
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:170]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 11]
```diff
@@ -159,7 +159,26 @@
...
+	
+	// create table for asigning posts
+	$stmt = $sqlConn->prepare("CREATE TABLE IF NOT EXISTS $assignPostTable ($postTitle VARCHAR(100) NOT NULL, $assignedUser VARCHAR(100), INDEX idx USING BTREE ($postTitle))");
+	if ( ! $stmt->execute() )
+	{
+		echo"there is a problem with database<br>";
+		die ();
+	}
+	
+	$stmt->close();
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:171]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 12]
```diff
@@ -159,7 +159,26 @@
...
+
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:172]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 13]
```diff
@@ -159,7 +159,26 @@
...
+	$stmt = $sqlConn->prepare("CREATE TABLE IF NOT EXISTS $assignPostTable ($postTitle VARCHAR(100) NOT NULL, $assignedUser VARCHAR(100), INDEX idx USING BTREE ($postTitle))");
+	if ( ! $stmt->execute() )
+	{
+		echo"there is a problem with database<br>";
+		die ();
+	}
+	
+	$stmt->close();
+	
+	// asign post ''
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:173]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 14]
```diff
@@ -159,7 +159,26 @@
...
+	$stmt2 = $sqlConn->prepare("INSERT INTO $assignPostTable VALUES (?,\"\")");
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:174]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 15]
```diff
@@ -159,7 +159,26 @@
...
+	$stmt2->bind_param ('s', $post );
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:175]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 16]
```diff
@@ -159,7 +159,26 @@
...
+	if ( ! $stmt2->execute() )
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:176]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected 0 spaces, found 4
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 5

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 17]
```diff
@@ -159,7 +159,26 @@
...
+	{
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:177]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 18]
```diff
@@ -159,7 +159,26 @@
...
+		echo"there is a problem with database<br>";
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:178]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 19]
```diff
@@ -159,7 +159,26 @@
...
+		die ();
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:179]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 20]
```diff
@@ -159,7 +159,26 @@
...
+	}
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:180]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected 0 spaces, found 4

File: includes/Post/HandleFinalDataOfCreatePost.php [hunk 21]
```diff
@@ -159,7 +159,26 @@
...
+	$stmt2->close();
```
Comment (pkgodara) [inline includes/Post/HandleFinalDataOfCreatePost.php:181]: __[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

**Comment by pkgodara** [comment]
Click here if you want to create code_sniffer's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/80422762/issues/pull_request/45/diff/1aa8d69ee072212342756f3c77be592b2b47f84c?open_modal=1#code_sniffer)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-code_sniffer)

**Review by NabeelQaiser — APPROVED** [review]

**Pull request merged**